Whisky/Views/Bottle/Pins/PinCreationView.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  PinCreationView.swift
+//  Whisky
+//
+//  This file is part of Whisky.
+//
+//  Whisky is free software: you can redistribute it and/or modify it under the terms
+//  of the GNU General Public License as published by the Free Software Foundation,
+//  either version 3 of the License, or (at your option) any later version.
+//
+//  Whisky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+//  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//  See the GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License along with Whisky.
+//  If not, see https://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UniformTypeIdentifiers
+import WhiskyKit
+
+struct PinCreationView: View {
+    var bottle: Bottle
+    @State var newPinName: String = ""
+    @State var newPinURL: URL = BottleData.defaultBottleDir
+    @State var pinPath: String = ""
+    @State var nameValid: Bool = false
+    @State var didError: Bool = false
+    @State var errorMessage: String = ""
+    @Environment(\.dismiss) var dismiss
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text("pin.title")
+                    .bold()
+                Spacer()
+            }
```

Comment:
We could probably get rid of all these wrapping `HStack`s and use `VStack(alignment: .leading)`

Reply:
Removed extraneous `HStack`s by utilizing this recommendation in the latest commit.